packages/cli/src/services/orchestration.service.ts

```diff
@@ -63,7 +64,13 @@ export class OrchestrationService {
 	async init() {
 		if (this.isInitialized) return;
 
-		if (config.get('executions.mode') === 'queue') await this.initPublisher();
+		if (config.get('executions.mode') === 'queue') {
```

Comment:
Is the `OrchestrationService` used in other execution modes as well?

Reply:
In theory yes, but the orchestration service is a bit of a tangle so I'm not confident to confirm. Tracking [here](https://linear.app/n8n/issue/PAY-1974/improvements-to-pubsub-setup).